File: JavaScript/HighestAndLowest.js
function highAndLow(numbers){
  const arrNumbers = numbers.split(" ").map((num) => +num)
  const maxNumber = Math.max(...arrNumbers);
  const minNumber = Math.min(...arrNumbers);
  return `${maxNumber} ${minNumber}`
}

console.assert(highAndLow("1 2 3") === "3 1", 'Erro no test 1');
console.assert(highAndLow("8 3 -5 42 -1 0 0 -9 4 7 4 -4") === "42 -9", 'Erro no test 2');

File: Python/Highest_and_Lowest.py
def high_and_low(numbers):
    arr_numbers = numbers.split(" ")
    max_number = max(list(map(int, arr_numbers)))
    min_number = min(list(map(int, arr_numbers)))
    
    return f"{max_number} {min_number}"


if __name__ == "__main__":
    print(high_and_low("-9 1 2 3"))
